style.css

```diff
@@ -53,15 +54,25 @@ body {
 .navigation-right {
   display: flex;
   align-items: center;
-  gap: 20px;
+  gap: 15px;
 }
 
 .navigation-right-cart {
-  width: 25px;
+  width: 18px;
+}
+
+.navigation-count {
+  font-size: 15px;
+  font-weight: bold;
```

Comment:
Why did you decide to use `bold ` here and not `700 ` as before? Better to use just one method